python/private/hermetic_runtime_repo_setup.bzl

```diff
@@ -44,13 +45,17 @@ def define_hermetic_runtime_toolchain_impl(
         python_version: {type}`str` The Python version, in `major.minor.micro`
             format.
         python_bin: {type}`str` The path to the Python binary within the
-            repositoroy.
+            repository.
         coverage_tool: {type}`str` optional target to the coverage tool to
             use.
+        free_threading: {type}`bool` optional free-threading support.
+            Default, False.
     """
     _ = name  # @unused
     version_info = semver(python_version)
     version_dict = version_info.to_dict()
+    version_dict["ft_postfix"] = "t" if free_threading else ""
```

Comment:
Thinking out loud, I am wondering if it makes sense to define a config setting using the free threading arg here and just include the files conditionally. That way we don't need to pass an extra `free_threading` bool arg to this macro, which could make things easier to maintain.